skaffold/src/main/kotlin/com/google/container/tools/skaffold/SkaffoldFiles.kt

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.container.tools.skaffold
+
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.vfs.VirtualFile
+import com.intellij.psi.search.FileTypeIndex
+import com.intellij.psi.search.GlobalSearchScope
+import org.jetbrains.yaml.YAMLFileType
+import java.io.ByteArrayInputStream
+import java.io.InputStream
+import java.util.Scanner
+import java.util.regex.Pattern
+
+// see https://github.com/GoogleContainerTools/skaffold/blob/master/examples/annotated-skaffold.yaml
```

Comment:
nit: should this have a block comment style (`/* */`). I'm not certain myself

Reply:
Didn't find anything in style guide. Maybe @tiembo knows some doc suggestions?